python/gauss2d/fit/parameters.cc

```diff
@@ -71,16 +72,16 @@ void bind_parameters(py::module &m) {
     declare_sizeparameter<double, g2f::SigmaYParameter, g2f::SizeYParameter>(m, "SigmaY");
     declare_parameter<double, g2f::StdDevParameter>(m, "StdDev");
     declare_transform_base<double>(m);
-    declare_transform<double, parameters::UnitTransform<double>>(m, "Unit");
+    declare_transform<double, modelfit_parameters::UnitTransform<double>>(m, "Unit");
     declare_transform<double, g2f::InverseTransform>(m, "Inverse");
     declare_transform<double, g2f::JanskyToABMagTransform>(m, "JanskyToABMag");
     declare_transform<double, g2f::NanojanskyToABMagTransform>(m, "NanojanskyToABMag");
     declare_transform<double, g2f::LogTransform>(m, "Log");
     declare_transform<double, g2f::Log10Transform>(m, "Log10");
     declare_transform<double, g2f::LogitTransform>(m, "Logit");
-    // TODO: Determine why this won't work with std::shared_ptr<parameters::Limits<double>>
+    // TODO: Determine why this won't work with std::shared_ptr<modelfit_parameters::Limits<double>>
     declare_transform_full<double, g2f::LogitLimitedTransform, true, true,
-                           std::shared_ptr<parameters::Limits<double>>, double>(m, "LogitLimited");
+                           std::shared_ptr<modelfit_parameters::Limits<double>>, double>(m, "LogitLimited");
```

Comment:
Why are you [requiring that the `Limits` be held by shared pointer](https://developer.lsst.io/cpp/style.html#b-smart-pointers-such-as-shared-ptr-should-only-be-used-as-arguments-if-a-reference-const-reference-or-raw-pointer-cannot-be-used)?

Reply:
This is a template parameter, not a function argument.

I did clarify the comment above because I'm not sure why it has to be done this way, but `declare_transform<double, g2f::LogitLimitedTransform, true, true>` does not compile.

Note that the `shared_ptr` there is ultimately used as the owning type in pybind11, it has to be either that or `unique_ptr`. For better or worse, I chose `shared_ptr` pretty much everywhere in this package, because even in the cases where I preferred `unique_ptr` it proved problematic to force all usages to unique in Python.

Reply:
Oops, didn't notice this was pybind11. Never mind.